python/sparkdl/transformers/tf_image.py

```diff
@@ -25,7 +25,7 @@
 from sparkdl.transformers.param import (
     keyword_only, HasInputCol, HasOutputCol, SparkDLTypeConverters)
 import sparkdl.transformers.utils as utils
-from sparkdl.utils import list_to_vector_udf
+import sparkdl.utils.udf
```

Comment:
Travis found more more thing :). I think you meant to do `import sparkdl.utils.udf as udf` here because you use the module on L239. But that won't actually work because you're shadowing the udf from `pyspark.sql.functions` imported a few lines above. Maybe we should rename this module?